tools/project_gen/project_gen/project.py

```diff
@@ -318,7 +318,7 @@ def generate_dbs(
         "--z-file",
         str(z_ffp),
     ]
-    ds_timeout = (n_stations * (60 * 60)) / (min(n_procs - 1, n_stations))
+    ds_timeout = (n_stations * (3600 * 24)) / (min(n_procs - 1, n_stations))
```

Comment:
This is probably too high? What's the expected time for one station to calculate?